### ISSUE TITLE
Data not being plotted/fitted correctly?

### ISSUE THREAD

**Issue by robineappen** [aleksandrbazhin/ellipsoid_fit_python#10]
Hi,

The code doesn't seem to produce the right result or maybe I'm looking at it the wrong way. I have attached a plot of the result. 

![chk1](https://user-images.githubusercontent.com/24272711/88063877-d5abcf00-cb6a-11ea-852f-b61520254db7.png)

The data seems to plot an oblate spheroid but the fit model (the orange plot) shows a prolate spheroid. Why is that? Is it because it is reading the wrong coordinates maybe?

[coordextract1_01259.txt](https://github.com/aleksandrbazhin/ellipsoid_fit_python/files/4953900/coordextract1_01259.txt)

I have attached the coordinate text file as well. Please let me know if you get the same result or not.

Thank you
Robin

**Comment by aleksandrbazhin** [comment]
The red one is actually a sphere - two days ago i fixed incompatibility with newer matplotlib #9 - they removed option for equal axes scale. The red dots are just the green ones with transformation which makes the green fitted ellipse into an orange sphere.
You can install 2.x version of matplotlib and uncomment lines in plot_ellipsoid.py [hack for equal axes](https://github.com/aleksandrbazhin/ellipsoid_fit_python/blob/f25413a5a949fc48353edb70cf0d21aacfd402b6/plot_ellipsoid.py#L28)

Maybe I should rewrite readme and add fixed version of matplotlib to requirements.txt, I don't know.
Also are you sure your data belong on the ellipsoid surface? If you are not sure, please, take a look at #5

**Comment by aleksandrbazhin** [comment]
![Figure_1-4](https://user-images.githubusercontent.com/875308/88134443-f9811a80-cc27-11ea-804f-edc576493ccf.png)

That is your plot stretched to fullhd and rotated a bit. I think it is a sphere, just uneven scale on axes is confusing now.

**Comment by aleksandrbazhin** [comment]
![Figure_1-5](https://user-images.githubusercontent.com/875308/88135009-634df400-cc29-11ea-9971-0a041e3aaabc.png)
And here is top-view, Which is perfectly correct for data not belonging on the surface of an ellipse. I think, I'll post an explanation as a separate issue.

**Comment by aleksandrbazhin** [comment]
Take a look at #11

**Comment by robineappen** [comment]
@aleksandrbazhin Yes, that was really helpful. I'll try to refine the data before using and will update here.
Thank you
